### DISCUSSION TITLE
next release of spline

### DISCUSSION THREAD

**Discussion by juja76** [AbsaOSS/spline#1368] [Q&A]
good morning,

recently you have release v2.2 of spark agent and I'm just wondering when spline with the latest producer's/consumer's modules will be released? and if in the meantime we can have access to the snapshot builds of those packages? I couldn't find it in any of the maven repos...

Thank you for your efforts and good luck!

**Comment by wajda** [comment]
Yes, you are right, it's been quite some time since the last Spline server release. The development was slow, but the version 1.0.0 was close to be released in spring 2023. But then the project was put on hold by the company due to lack of resources and only recently it again started gaining some attention from the business. I'm currently finishing a custom DB transaction recovery mechanism, then we need a new export/import functionality, then some testing, and I hope the release will be completed by the end of September at the latest.

**Reply by juja76** [reply]
got it, thank you! can we access a snapshot builds in the meantime, if there are any?

**Reply by wajda** [reply]
The snapshot builds are not stored. You need to build the project from sources.
```
mvn install -Ddocker -Ddocker.repositoryUrl=my
```
or if you don't need a docker then just:
```
mvn install
```
To disable tests add `-DskipTests`
